src/gsi/gsi_rfv3io_mod.f90

```diff
@@ -640,13 +680,16 @@ subroutine read_fv3_netcdf_guess(fv3filenamegin)
     integer(i_kind) k,i,j
     integer(i_kind) it,ier,istatus
     real(r_kind),dimension(:,:),pointer::ges_ps=>NULL()
+    real(r_kind),dimension(:,:),pointer::ges_ps_readin=>NULL()
     real(r_kind),dimension(:,:),pointer::ges_z=>NULL()
     real(r_kind),dimension(:,:,:),pointer::ges_u=>NULL()
     real(r_kind),dimension(:,:,:),pointer::ges_v=>NULL()
     real(r_kind),dimension(:,:,:),pointer::ges_q=>NULL()
 !   real(r_kind),dimension(:,:,:),pointer::ges_ql=>NULL()
```

Comment:
Since it is still commented out, please remove commented out line unless it will be used in the future.